x-pack/protocol/src/main/java/org/elasticsearch/protocol/xpack/rollup/job/RollupJobConfig.java

```diff
@@ -119,7 +131,7 @@ public RollupJobConfig(final String id,
         // Cron doesn't have a parse helper method to see if the cron is valid,
         // so just construct a temporary cron object and if the cron is bad, it'll
         // throw an exception
-        Cron testCron = new Cron(cron);
+        //Cron testCron = new Cron(cron);
```

Comment:
Note: here `Cron` is used to validate the cron expression when instanciating the `RollupJobConfig`. Maybe this validation could be done on the server side (transport action)? Or should we bring `Cron` in the common lib?

Reply:
For ML we had the same problem only much, much worse: to reproduce all our server-side config validation in the HLRC would mean moving several classes you'd expect to be server-side only into the protocol library.  We decided not to do complex validation client-side to avoid moving such classes.

Reply:
++ to moving this "validation" to the transport action, and adding a TODO to make an actual validation method at some point instead of this hack.

Reply:
Thanks, I moved the validation to the transport action